src/SimpleRADOSStriper.cc

```diff
@@ -488,12 +488,17 @@ ssize_t SimpleRADOSStriper::read(void* data, size_t len, uint64_t off)
   }
 
   size_t r = 0;
-  std::vector<std::pair<bufferlist, aiocompletionptr>> reads;
+  std::vector<std::pair<blptr, aiocompletionptr>> reads;
```

Comment:
ah, so the `bufferlist`s are being moved whenever the vector resizes. maybe switch to a `std::deque` or `list` instead?

Reply:
(`std::deque` doesn't invalidate references on back-insertion, and would do fewer allocations than this `unique_ptr` approach)

Reply:
@rzarzynski